src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts

```diff
@@ -26,8 +26,8 @@ import { applyZoom, zoomIn, zoomOut } from 'vs/platform/window/electron-sandbox/
 import { StandardKeyboardEvent } from 'vs/base/browser/keyboardEvent';
 import { KeyCode } from 'vs/base/common/keyCodes';
 
-const DEBUG_FLAGS_PATTERN = /\s--(inspect|debug)(-brk|port)?=(\d+)?/;
-const DEBUG_PORT_PATTERN = /\s--(inspect|debug)-port=(\d+)/;
+const DEBUG_FLAGS_PATTERN = /\s--inspect(?:-brk|port)?=(\d+)?/;
+const DEBUG_PORT_PATTERN = /\s--inspect-port=(\d+)/;
```

Comment:
Should `-port` be removed from the first one?

Reply:
no, `--inspect-port` is still allowed. I had to check on that 😛

Reply:
Oh I misread the first one, it's `--inspectport` and `--inspect-port`.